README.rst

```diff
@@ -53,27 +53,37 @@ From source:
     python setup.py install
 
 
+Why Ignite?
+===========
+Ignite's high level of abstraction assumes less about the type of network (or networks) that you are training, and we require the user to define the closure to be run in the training and validation loop. This level of abstraction allows for a great deal more of flexibility, such as co-training multiple models (i.e. GANs) and computing/tracking multiple losses and metrics in your training loop.
+
+Ignite also allows for multiple handlers to be attached to events, and a finer granularity of events in the engine loop.
+
+
 Documentation
 =============
 API documentation and an overview of the library can be found `here <https://pytorch.org/ignite/index.html>`_.
 
+
+Structure
+=========
+- **ignite**: Core of the library, contains an engine for training and evaluating, all of the classic machine learning metrics and a variety of handlers to ease the pain of training and validation of neural networks! 
+
+**ignite.contrib**: The Contrib directory contains additional modules contributed by Ignite users. Modules vary from TBPTT engine, various optimisation parameter schedulers, logging handlers and a metrics module containing many regression metrics (`ignite.contrib.metrics.regression <https://github.com/pytorch/ignite/tree/master/ignite/contrib/metrics/regression>`_)! 
+
+The code in **ignite.contrib** is not fully maintained as the core part of the library. It may change or be removed at any time without notice.
```

Comment:
nit: "is not fully maintained" -> "is not as fully maintained"

Reply:
Incorporated!